server/templates/home.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+<div id="page"></div>
```

Comment:
following from that hackernoon article -- would it make sense to have multiple react mount points for different page sections? E.g., set up the header to always mount at `<div id="header">", and then have a thing like:
```
<div id="page-content">
        <!-- Contents get replaced by mounted React.Component -->
        <i class="fa fa-lg fa-spinner fa-spin"></i><br><br>
        <i class="pending">Loading components...</i><br><br>
</div>
```

which they use to show a spinner and then be the mounting point for the main page content?

I don't know if this would be better, or if it's preferable to mount everything from a single mount point. (mostly i'm just trying to understand how this works, so i'm trying to ask some questions to see if i'm getting it at all :-) )

Reply:
Hmm, had to think about this a bit. I'm pretty sure that functionally the approaches would be similar in terms of behavior. But intuitively I think I prefer it where each "view" is its own top-level component and all the composition happens on the client side. It's a little bit simpler and easier to understand.